packages/devextreme/js/__internal/grids/grid_core/views/m_rows_view.ts

```diff
@@ -45,6 +45,16 @@ function isGroupRow({ rowType, column }) {
     && !column.showWhenGrouped && !column.command;
 }
 
+function watch({
+  element, watch, getter, callBack,
+}) {
```

Comment:
Let's rename function name or argument name. Or maybe inline this function
It's unsafe to have the same name for func and for variable inside of it, there was a case when we needed to rename it because of some issues in demos: https://github.com/DevExpress/DevExtreme/pull/25850/